dev-docs/publisher-api-reference.md

```diff
@@ -613,39 +613,48 @@ See the table below for the list of properties in the `mediaTypes` object of the
 For an example of a native ad unit, see below.  For more detailed instructions, see [Show Native Ads]({{site.baseurl}}/dev-docs/show-native-ads.html).
 
 ```javascript
-pbjs.addAdUnits({
-    code: slot.code,
-    sizes: slot.size,
-    mediaTypes: {
-        native: {
-            image: {
-                required: true
-            },
-            title: {
-                required: true,
-                len: 80
-            },
-            sponsoredBy: {
-                required: true
-            },
-            clickUrl: {
-                required: true
-            },
-            body: {
-                required: true
-            },
-            icon: {
-                required: true
+    pbjs.addAdUnits({
+        code: slot.code,
+        sizes: slot.size,
+        mediaTypes: {
+            native: {
+                image: {
+                    required: true,
+
+                    /* If your bidder supports it, you can specify
+                       aspect ratios for image and icon native assets */
+
+                    aspect_ratios: [{
```

Comment:
I think our basic native examples should show "mediaTypes.native.image.sizes", as this will be the most widely supported by bidder adapters.  

Separately, we should document the following structures as the two possible methods for defining sizes for image-like assets:

- mediaTypes.native.image.sizes
- mediaTypes.native.image.aspect_ratios

Reply:
@mjacobsonny thanks for the review - I think this comment (and the below) are addressed by my changes in 635016e.

Mind taking another look?